tests/unit/checkmatelib/client_test.py

```diff
@@ -69,10 +70,35 @@ def test_it_truncates_very_long_urls(self, client, requests):
         assert len(called_url) == client.MAX_URL_LENGTH
         assert called_url == very_long_url[: client.MAX_URL_LENGTH]
 
+    def test_send_auth_api_key(self, auth_client, requests):
+        auth_client.check_url("http://bad.example.com")
+
+        requests.get.assert_called_once_with(
+            "http://checkmate.example.com/api/check",
+            params={"url": "http://bad.example.com"},
+            timeout=1,
+            auth=("SECRET",),
+        )
```

Comment:
We could tighten up this assert a little to only assert what this particular test cares about:

```suggestion
assert requests.get.call_args[1]["auth"] == ("SECRET",)
```